packages/harmony/src/foundations/corner-radius/cornerRadius.ts

```diff
@@ -4,7 +4,8 @@ export const cornerRadius = {
   m: 8,
   l: 12,
   xl: 16,
-  '2xl': 32
+  '2xl': 32,
+  '3xl': 48
```

Comment:
big boy curves. 

Not sure if there have already been convos about it but if we're adding this on the dev side we should add it on the design side as well 👍  @szonana @julianbaker

Reply:
yes thank you for callout <3